### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Mihndim2020**
This is the first version of our responsive design(week two project) project. It is a clone of thenextweb site. The content is not exactly the same, we are trying to implement the responsiveness of the site.

### CODE REVIEW

**Comment by Mihndim2020** [comment]
Please I have one stubborn stylelint error. Please I need a hint on how to make it go away. Please do have a look.

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,582 @@
...
+  float: left;
+  padding: 15px;
+  width: 100%;
+}
+
+/* this is for mobile phones  */
+
+/* [class*="col-"] {
+  width: 100%;
+} */
```
Comment (andresporras3423) [inline css/style.css]: Remove commented code

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,582 @@
...
+    z-index: 5;
+  }
+
+  .sticky-nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    z-index: 5;
+  }
```
Comment (andresporras3423) [inline css/style.css:332]: Group selectors with the same style definition

File: index.html [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.1/css/all.css"
+      integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp"
+      crossorigin="anonymous"
+    />
+    <!-- <script
+      type="text/javascript"
+      src="https://api.iconify.design/entypo-social.js?icons=instagram"
+      async="true"
+    ></script> -->
```
Comment (andresporras3423) [inline index.html]: Remove commented code

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Julius Mih** 
You did a great job with your first submission for this project, your page looks really similar to the original  :clap: :clap:
There are still some improvements to do:

- [ ] Your project should pass all the linter checks
- [ ] In the README file, the links to your screenshots are broken
- [ ] Remove horizontal scrollbar
- [ ] TWN logo should disappear in medium and small screen size:

| Yours | Original |
|:---:|:---:|
|image | image |

- [ ] Latest funding section should have three articles per row in medium screen size

| Yours | Original |
|:---:|:---:|
|image | image |

- [ ] The categories section is missing
- [ ] Remove commented code from the index.html file
- [ ] Remove commented code from the style.css file
- [ ] Group selectors with the same style definition 
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

File: css/style.css (hunk 1 above)
Comment (Mihndim2020) [reply css/style.css]: commented code removed

File: css/style.css (hunk 2 above)
Comment (Mihndim2020) [reply css/style.css:332]: selectors with same style definition grouped

File: index.html (hunk 3 above)
Comment (Mihndim2020) [reply index.html]: commented code removed

**Comment by Mihndim2020** [comment]
All the requested changes effected, except for the single linter error, which seems to a system configuration file issue. Please we do need help with that. Thank you. 
linter-erro

**Comment by Mihndim2020** [comment]
Requested changes effected as requested, but we still have an issue with a single linter error. When we run the linters locally, everything works fine, but when we upload and do a pull request, that error appears. We are tempted to thing it is a configuration error which is beyond the scope of the project. We really do need help on that. Below is a screenshot. Thanks for helping out.
linter-erro

**Comment by Mihndim2020** [comment]
Requested changes effected as requested, but we still have an issue with a single linter error. When we run the linters locally, everything works fine, but when we upload and do a pull request, that error appears. We are tempted to thing it is a configuration error which is beyond the scope of the project. We really do need help on that. Below is a screenshot. Thanks for helping out.
linter-erro

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi @Mihndim2020,  I am  [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏👏   I appreciate your hard work in replicating thenextweb.com. Kindly fix the following issues. 

[REQUIRED]
- [ ] Fix stylelint linting errors.

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Comment by Mihndim2020** [comment]
Requested changes effected and linters  errors fixed

**Review by layzrd — APPROVED** [review]
### Status: Approved 🙌 🎇
Hi @Mihndim2020, I am  [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  Congratulations 🎊 👏, you did very well on this project.

Now, you can move ahead.

Please don't forget to merge to master. Good luck with the next projects!

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Comment by Mihndim2020** [comment]
Done and dusted !

**Pull request merged**